### ISSUE TITLE
Attention mask

### ISSUE THREAD

**Issue by ZXMMD** [PKU-YuanGroup/Open-Sora-Plan#397]
关键参数设置：
train_batch_size: 1
num_frames: 29
max_height: 360
max_width: 640
train_fps: 12
ae: CausalVAEModel_D4_4x8x8
model:  OpenSoraT2V-ROPE-L/122
text_encoder_name:  google/mt5-xxl

只是换了我自己的数据集，没有使用LengthGroupedSampler。没有修改其他代码。
运行：bash scripts/text_condition/gpu/train_t2v.sh
会在 [dataset_utils.py](https://github.com/PKU-YuanGroup/Open-Sora-Plan/blob/main/opensora/utils/dataset_utils.py)中的下列代码报错：
```python
if self.batch_size == 1 or self.group_frame or self.group_resolution:
    assert torch.all(attention_mask.bool())
```
打印attention mask,确实不全为1：
<img width="291" alt="image" src="https://github.com/user-attachments/assets/66e49f1b-1e25-4da1-bb40-8e353d95d53f">
请问您知道问题可能出在哪里吗？感谢！

**Comment by ZXMMD** [comment]
求解答🙏

**Comment by ZXMMD** [comment]
明白了，因为max_height（360）不是16的整数倍
